tests/test_amclient.py

```diff
@@ -10,6 +10,11 @@
 import unittest
 import uuid
 
+try:
+    from unittest import mock
+except ImportError:
+    import mock
```

Comment:
I think that it's safe just to use `import mock`.

Reply:
done.

Reply:
Great to know. Thanks for trying!